[On Hold] Add Heliarch and Resistance intros (#4800)
endless-sky/endless-sky: data/coalition/coalition jobs.txt

		payment
		payment 25000
		dialog phrase "coalition games dialog"

mission "Hely Contain True"
	job
	repeat
	name `Contain Resistance`
	description `Fly to <destination> so that you and <bunks> other Heliarch agents can contain a Resistance outbreak.`
	deadline
	passengers 8 23
	to offer

> 		random < 15

Author: upped their randoms

		has "license: Heliarch"
	source
		government "Heliarch"
	stopover

> 		distance 6 10

Author: idk why it even was a stopover, changed it to `destination`, added `not attributes "coalition station"` (hoping that works) to prevent it from rolling those as a destination

	on offer
		require "Enforcer Confrontation Gear"
	on visit
		dialog phrase "generic passenger on visit"

> 	on complete

Author: shouldn't be a problem anymore since it was changed to a destination

	on complete
		payment
		payment 90000
		dialog `The agents thank you for transporting them in your ship before wishing you safe travels, heading off to deliver your findings to their superiors. You're transferred <payment> a few moments later.`

mission "Evacuate Resistance True"
	name "Rescue Resistance Members"
	description "Pick up the <bunks> Resistance members on <stopovers>, then bring them to <destination>."
	deadline
	repeat
	minor
	to offer
		random < 3
		has "Heliarch Interrogation 2: declined"
		has "pledged to resistance"
		not "Evacuate Resistance True: active"
		not "Evacuate Resistance Fake: active"
		not "assisted heliarch"
		not "Heliarch Investigation 2: active"
		not "Heliarch Containment 4-A: offered"
	to fail
		has "Heliarch Investigation 2: active"
	passengers 24 65
	source
		government "Coalition"
	stopover
		government "Coalition"
		distance 2 6
	destination
		government "Coalition"
		distance 3 5
	on offer

> 		dialog `You wander around the spaceport for a bit, without finding anything of interest. Once you get back to your ship, you receive a message from the Resistance: <bunks> members need someone to rescue them on <stopovers>, where the Heliarchs are investigating, and transport them to <destination>.`

Reviewer: ```suggestion
		dialog `You wander around the spaceport for a bit without finding anything of interest. Once you get back to your ship, you receive a message from the Resistance: <bunks> members need someone to rescue them on <stopovers>, where the Heliarchs are investigating, and transport them to <destination>.`
```

	on stopover
		dialog `The Resistance members rush inside your ship, urging you to take off quickly.`
	on visit
		dialog phrase "generic missing stopover or passengers"
	on complete
		dialog `The Resistance members thank you for rescuing them before handing you <payment>.`

> 		payment 23861 8

Reviewer: I don't know a lot about syntax here, but the space seems strange. Wanted to point it out and confirm it was intentional.

mission "Evacuate Resistance Fake"
	name "Rescue Resistance Members"
	description "Pick up the <bunks> Resistance members on <stopovers>, then bring them to <destination>."
	deadline
	repeat
	minor
	to offer
		random < 3
		has "Heliarch Interrogation 2: declined"
		has "pledged to resistance"
		not "Evacuate Resistance True: active"
		not "Evacuate Resistance Fake: active"
		not "assisted heliarch"
		not "Heliarch Investigation 2: active"
		not "Heliarch Containment 4-A: offered"
	to fail
		has "Heliarch Investigation 2: active"
	passengers 24 65
	source
		government "Coalition"
	stopover
		government "Coalition"
		distance 2 6
	destination
		government "Coalition"
		distance 3 5
	on offer

> 		dialog `You wander around the spaceport for a bit, without finding anything of interest. Once you get back to your ship, you receive a message from the Resistance: <bunks> members need someone to rescue them on <stopovers>, where the Heliarchs are investigating, and transport them to <destination>.`

Reviewer: ```suggestion
		dialog `You wander around the spaceport for a bit without finding anything of interest. Once you get back to your ship, you receive a message from the Resistance: <bunks> members need someone to rescue them on <stopovers>, where the Heliarchs are investigating, and transport them to <destination>.`
```

	on stopover

> 		dialog `You land at the meeting spot, waiting for the Resistance members to arrive. As time passes, and no one shows up, it's clear that the Heliarchs got to them before you did.`

Reviewer: ```suggestion
		dialog `You land at the meeting spot, waiting for the Resistance members to arrive. As time passes and no one shows up, it's clear that the Heliarchs got to them before you did.`
```

		fail

mission "Resistance: Armageddon Core Delivery"
	name "Armageddon Core Delivery"

> 	description "Resistance members are offering <payment> for you to deliver an Armageddon Core to <destination>"

Reviewer: ```suggestion
	description "Resistance members are offering <payment> for you to deliver an Armageddon Core to <destination>."
```

	repeat
	minor
	to offer
		random < 4
		has "pledged to resistance"
		has "Resistance: Reactors: done"
		not "Resistance: Armageddon Core Delivery: active"
		not "assisted heliarch"
		not "Heliarch Investigation 2: active"
		not "Heliarch Containment 4-A: offered"
	to fail
		has "Heliarch Investigation 2: active"
	source
		government "Coalition"
	destination
		distance 5 12
		not government "Heliarch"
	on offer

> 		dialog `As you enter the spaceport, you're dragged away to an isolated part by someone who claims to be a Resistance member. They speak briefly, saying, "Comrades in <destination> have requested an Armageddon Core."`

Reviewer: ```suggestion
		dialog `As you enter the spaceport, you're dragged away to an isolated part by someone who claims to be a Resistance member. They speak briefly, saying, "Comrades on <destination> have requested an Armageddon Core."`
```

	on visit
		dialog phrase "generic cargo on visit"
	on complete
		"supplied resistance" ++
		outfit "Armageddon Core" -1
		payment 9000000

> 		dialog `Some Resistance members unload the Armageddon Core after you land on an isolated part of <planet>, as instructed. They hand you <payment> before quietly transporting the Core away.`

Reviewer: ```suggestion
		dialog `Some Resistance members unload the Armageddon Core after you land on an isolated part of <planet> as instructed. They hand you <payment> before quietly transporting the Core away.`
```

mission "Resistance: Fusion Reactor Delivery"
	name "Fusion Reactor Delivery"

> 	description "Resistance members are offering <payment> for you to deliver a Fusion Reactor to <destination>"

Reviewer: ```suggestion
	description "Resistance members are offering <payment> for you to deliver a Fusion Reactor to <destination>."
```

	repeat
	minor
	to offer
		random < 7
		has "pledged to resistance"
		has "Resistance: Reactors: done"
		not "Resistance: Fusion Reactor Delivery: active"
		not "assisted heliarch"
		not "Heliarch Investigation 2: active"
		not "Heliarch Containment 4-A: offered"
	to fail
		has "Heliarch Investigation 2: active"
	source
		government "Coalition"
	destination
		distance 3 8
		not government "Heliarch"
	on offer

> 		dialog `As you enter the spaceport, you're dragged away to an isolated area by someone who claims to be a Resistance member. They speak briefly, saying, "Comrades in <destination> have requested a Fusion Reactor."`

Reviewer: ```suggestion
		dialog `As you enter the spaceport, you're dragged away to an isolated area by someone who claims to be a Resistance member. They speak briefly, saying, "Comrades on <destination> have requested a Fusion Reactor."`
```

	on visit
		dialog phrase "generic cargo on visit"
	on complete
		"supplied resistance" ++
		outfit "Fusion Reactor" -1
		payment 5000000

> 		dialog `Some Resistance members unload the Fusion Reactor after you land on an isolated part of <planet>, as instructed. They hand you <payment> before quietly transporting the Reactor away.`

Reviewer: ```suggestion
		dialog `Some Resistance members unload the Fusion Reactor after you land on an isolated part of <planet> as instructed. They hand you <payment> before quietly transporting the Reactor away.`
```

mission "Resistance: Breeder Reactor Delivery"
	name "Breeder Reactor Delivery"

> 	description "Resistance members are offering <payment> for you to deliver a Breeder Reactor to <destination>"

Reviewer: ```suggestion
	description "Resistance members are offering <payment> for you to deliver a Breeder Reactor to <destination>."
```

	repeat
	minor
	to offer
		random < 10
		has "pledged to resistance"
		has "Resistance: Reactors: done"
		not "Resistance: Breeder Reactor Delivery: active"
		not "assisted heliarch"
		not "Heliarch Investigation 2: active"
		not "Heliarch Containment 4-A: offered"
	to fail
		has "Heliarch Investigation 2: active"
	source
		government "Coalition"
	destination
		distance 3 6
		not government "Heliarch"
	on offer

> 		dialog `As you enter the spaceport, you're dragged away to an isolated area by someone who claims to be a Resistance member. They speak briefly, saying, "Comrades in <destination> have requested a Breeder Reactor."`

Reviewer: ```suggestion
		dialog `As you enter the spaceport, you're dragged away to an isolated area by someone who claims to be a Resistance member. They speak briefly, saying, "Comrades on <destination> have requested a Breeder Reactor."`
```

	on visit
		dialog phrase "generic cargo on visit"
	on complete
		"supplied resistance" ++
		outfit "Breeder Reactor" -1
		payment 2500000

> 		dialog `Some Resistance members unload the Breeder Reactor after you land on an isolated part of <planet>, as instructed. They hand you <payment> before quietly transporting the Reactor away.`

Reviewer: ```suggestion
		dialog `Some Resistance members unload the Breeder Reactor after you land on an isolated part of <planet> as instructed. They hand you <payment> before quietly transporting the Reactor away.`
```

mission "Resistance: Torpedo Delivery"
	name "Torpedo Delivery"

> 	description "Resistance members are offering <payment> for you to deliver a Torpedo Launcher, complete with ammunition, to <destination>."

Reviewer: ```suggestion
	description "Resistance members are offering <payment> for you to deliver a Torpedo Launcher, complete with 30 torpedoes, to <destination>."
```
It would be a good idea to clarify how much is needed for those buying to cargo.

	repeat
	minor
	to offer
		random < 12
		has "pledged to resistance"
		has "Resistance: Torpedoes: done"
		not "Resistance: Torpedo Delivery: active"
		not "assisted heliarch"
		not "Heliarch Investigation 2: active"
		not "Heliarch Containment 4-A: offered"
	to fail
		has "Heliarch Investigation 2: active"
	source
		government "Coalition"
	destination
		distance 6 12
		not government "Heliarch"
	on offer

> 		dialog `As you enter the spaceport, you're dragged away to an isolated area by someone who claims to be a Resistance member. They speak briefly, saying, "Comrades in <destination> have requested a Torpedo Launcher and ammunition for it."`

Reviewer: ```suggestion
		dialog `As you enter the spaceport, you're dragged away to an isolated area by someone who claims to be a Resistance member. They speak briefly, saying, "Comrades on <destination> have requested a Torpedo Launcher and ammunition for it."`
```

	on visit
		dialog phrase "generic cargo on visit"
	on complete
		"supplied resistance" ++
		outfit "Torpedo Launcher" -1
		outfit "Torpedo" -30
		payment 122000

> 		dialog `Some Resistance members unload the Torpedo Launcher and it's ammunition after you land on an isolated part of <planet>, as instructed. They hand you <payment> before quietly transporting the cargo away.`

Reviewer: ```suggestion
		dialog `Some Resistance members unload the Torpedo Launcher and its ammunition after you land on an isolated part of <planet> as instructed. They hand you <payment> before quietly transporting the cargo away.`
```

mission "Resistance: Typhoon Delivery"
	name "Typhoon Delivery"

> 	description "Resistance members are offering <payment> for you to deliver a Typhoon Launcher, complete with ammunition, to <destination>."

Reviewer: ```suggestion
	description "Resistance members are offering <payment> for you to deliver a Typhoon Launcher, complete with 30 Typhoon Torpedoes, to <destination>."
```

	repeat
	minor
	to offer
		random < 7
		has "pledged to resistance"
		has "Resistance: Torpedoes: done"
		not "Resistance: Typhoon Delivery: active"
		not "assisted heliarch"
		not "Heliarch Investigation 2: active"
		not "Heliarch Containment 4-A: offered"
	to fail
		has "Heliarch Investigation 2: active"
	source
		government "Coalition"
	destination
		distance 6 12
		not government "Heliarch"
	on offer

> 		dialog `As you enter the spaceport, you're dragged away to an isolated area by someone who claims to be a Resistance member. They speak briefly, saying, "Comrades in <destination> have requested a Typhoon Launcher and ammunition for it."`

Reviewer: ```suggestion
		dialog `As you enter the spaceport, you're dragged away to an isolated area by someone who claims to be a Resistance member. They speak briefly, saying, "Comrades on <destination> have requested a Typhoon Launcher and ammunition for it."`
```

	on complete
		"supplied resistance" ++
		outfit "Typhoon Launcher" -1
		outfit "Typhoon Torpedo" -30
		payment 341000

> 		dialog `Some Resistance members unload the Typhoon Launcher and it's ammunition after you land on an isolated part of <planet>, as instructed. They hand you <payment> before quietly transporting the cargo away.`

Reviewer: ```suggestion
		dialog `Some Resistance members unload the Typhoon Launcher and its ammunition after you land on an isolated part of <planet> as instructed. They hand you <payment> before quietly transporting the cargo away.`
```

mission "Resistance: Plasma Delivery"
	name "Plasma Turret Delivery"
	description "Resistance members are offering <payment> for you to deliver a Plasma Turret to <destination>."
	repeat
	minor
	to offer
		random < 6
		has "pledged to resistance"
		has "Resistance: Plasma: done"
		not "Resistance: Plasma Delivery: active"
		not "assisted heliarch"
		not "Heliarch Investigation 2: active"
		not "Heliarch Containment 4-A: offered"
	to fail
		has "Heliarch Investigation 2: active"
	source
		government "Coalition"
	destination
		distance 4 9
		not government "Heliarch"
	on offer

> 		dialog `As you enter the spaceport, you're dragged away to an isolated area by someone who claims to be a Resistance member. They speak briefly, saying, "Comrades in <destination> have requested a Plasma Turret."`

Reviewer: ```suggestion
		dialog `As you enter the spaceport, you're dragged away to an isolated area by someone who claims to be a Resistance member. They speak briefly, saying, "Comrades on <destination> have requested a Plasma Turret."`
```

	on complete
		"supplied resistance" ++
		outfit "Plasma Turret" -1
		payment 520000

> 		dialog `Some Resistance members unload the Plasma Turret after you land on an isolated part of <planet>, as instructed. They hand you <payment> before quietly transporting the Turret away.`

Reviewer: ```suggestion
		dialog `Some Resistance members unload the Plasma Turret after you land on an isolated part of <planet> as instructed. They hand you <payment> before quietly transporting the turret away.`
```
`Plasma Turret` is a proper noun, `turret` is not.

mission "Resistance: Flamethrower Delivery"
	name "Flamethrower Delivery"
	description "Resistance members are offering <payment> for you to deliver a pair of Flamethrowers to <destination>."
	repeat
	minor
	to offer
		random < 8
		has "pledged to resistance"
		has "Resistance: Flamethrowers: done"
		not "Resistance: Flamethrower Delivery: active"
		not "assisted heliarch"
		not "Heliarch Investigation 2: active"
		not "Heliarch Containment 4-A: offered"
	to fail
		has "Heliarch Investigation 2: active"
	source
		government "Coalition"
	destination
		distance 2 6
		not government "Heliarch"
	on offer
		dialog `As you enter the spaceport, you're dragged away to an isolated area by someone who claims to be a Resistance member. They speak briefly, saying, "Comrades in <destination> have requested a pair of Flamethrowers."`
	on complete
		"supplied resistance" ++
		outfit "Flamethrower" -2
		payment 456000

> 		dialog `Some Resistance members unload the Flamethrowers after you land on an isolated part of <planet>, as instructed. They hand you <payment> before quietly transporting the Flamethrowers away.`

Reviewer: ```suggestion
		dialog `Some Resistance members unload the Flamethrowers after you land on an isolated part of <planet> as instructed. They hand you <payment> before quietly transporting the Flamethrowers away.`
```

mission "Resistance: Grenade Delivery"
	name "Fragmentation Grenade Delivery"
	description "Resistance Members are offering <payment> for you to deliver six Fragmentation Grenades to <destination>."
	repeat
	minor
	to offer
		random < 11
		has "pledged to resistance"
		not "Resistance: Grenade Delivery: active"
		not "assisted heliarch"
		not "Heliarch Investigation 2: active"
		not "Heliarch Containment 4-A: offered"
	to fail
		has "Heliarch Investigation 2: active"
	source
		government "Coalition"
	destination
		distance 7 13
		not government "Heliarch"
	on offer

> 		dialog `As you enter the spaceport, you're dragged away to an isolated area by someone who claims to be a Resistance member. They speak briefly, saying, "Comrades in <destination> have requested half a dozen Fragmentation Grenades."`

Reviewer: ```suggestion
		dialog `As you enter the spaceport, you're dragged away to an isolated area by someone who claims to be a Resistance member. They speak briefly, saying, "Comrades on <destination> have requested half a dozen Fragmentation Grenades."`
```

	on complete
		"supplied resistance" ++
		outfit "Fragmentation Grenades" -6
		payment 102000

> 		dialog `Some Resistance members unload the Fragmentation Grenades after you land on an isolated part of <planet>, as instructed. They hand you <payment> before quietly transporting the Grenades away.`

Reviewer: ```suggestion
		dialog `Some Resistance members unload the Fragmentation Grenades after you land on an isolated part of <planet> as instructed. They hand you <payment> before quietly transporting the grenades away.`
```
Same here with the difference between proper and normal nouns.

mission "Resistance: Anti-Missile Delivery"
	name "Anti-Missile Turret Delivery"
	description "Resistance Members are offering <payment> for you to deliver a Heavy Anti-Missile Turret to <destination>."
	repeat
	minor
	to offer
		random < 10
		has "pledged to resistance"
		has "Resistance: AM: done"
		not "Resistance: Anti-Missile Delivery: active"
		not "assisted heliarch"
		not "Heliarch Investigation 2: active"
		not "Heliarch Containment 4-A: offered"
	to fail
		has "Heliarch Investigation 2: active"
	source
		government "Coalition"
	destination
		distance 9 14
		not government "Heliarch"
	on offer

> 		dialog `As you enter the spaceport, you're dragged away to an isolated area by someone who claims to be a Resistance member. They speak briefly, saying, "Comrades in <destination> have requested a Heavy Anti-Missile Turret."`

Reviewer: ```suggestion
		dialog `As you enter the spaceport, you're dragged away to an isolated area by someone who claims to be a Resistance member. They speak briefly, saying, "Comrades on <destination> have requested a Heavy Anti-Missile Turret."`
```

	on complete
		"supplied resistance" ++
		outfit "Heavy Anti-Missile Turret" -1
		payment 150000

> 		dialog `Some Resistance members unload the Heavy Anti-Missile Turret after you land on an isolated part of <planet>, as instructed. They hand you <payment> before quietly transporting the Turret away.`

Reviewer: ```suggestion
		dialog `Some Resistance members unload the Heavy Anti-Missile Turret after you land on an isolated part of <planet> as instructed. They hand you <payment> before quietly transporting the turret away.`
```
Same case here as well.
